executor/mock/jobrunner.go

```diff
@@ -168,6 +167,14 @@ func NewJobRunner() *JobRunner {
 		panic(err)
 	}
 
+	return cfg, dockerCfg
+}
+
+// NewJobRunner creates a new JobRunner with its executor started
+// in the background and the test driver configured to use it.
+func NewJobRunner() *JobRunner {
```

Comment:
TODO: We should move the job runner into the module standalone, given it gets used nowhere else.